client/state/ui/importers/actions.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { IMPORTS_IMPORTER_OPTION_SELECT } from 'state/action-types';
+
+/**
+ * Selects a specific importer option from the available importers.
+ *
+ * @param  {String}  importerOption The selected importer option
+ * @return {Object}  Action
+ */
+export const selectImporterOption = importerOption => ( {
```

Comment:
I tried calling this manually like so:
`dispatch( { type: "IMPORTS_IMPORTER_OPTION_SELECT", importerOption: 'importer-type-site-importer' } )`

...and it's erring out...

```
Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: null.

Check the render method of `ImporterHeader`
```



This warning is probably helpful:
```
Warning: Failed prop type: The prop `importerStatus.importerState` is marked as required in `ImporterSiteImporter`, but its value is `undefined`.
```

If it's unsupported to trigger this action before some state is populated, we should handle that specifically.

If not, we should build some default state when none is present.

In either case, test coverage on actions would be desirable here.